docs/code-quality/fxcop-rule-port-status.md

```diff
@@ -25,7 +25,9 @@ Rule ID | Title
 --------|---------
 [CA1000](ca1000.md) | Do not declare static members on generic types
 [CA1001](ca1001.md) | Types that own disposable fields should be disposable
+[CA1002](ca1002.md) | Do not expose generic lists
```

Comment:
Tagging @Evangelink to verify the list as we he was the one who ported majority, if not all, of these rules.